web/src/modules/app/reducers/index.js

```diff
@@ -43,10 +42,10 @@ const endpointNames = [
   TUNEWS_ELEMENT_ENDPOINT_NAME
 ]
 
-export const startFetchReducer = <T: PayloadDataType> (oldPayload?: Payload<T>, action: StartFetchActionType<T>
-): Payload<T> => action.payload
+export const startFetchReducer = (oldPayload?: Payload<*>, action: StartFetchActionType<*>
+): Payload<*> => action.payload
 
-export const finishFetchReducer = <T: PayloadDataType> (oldPayload?: Payload<T>, action: FinishFetchActionType<T>
+export const finishFetchReducer = <T: PayloadDataType> (oldPayload?: Payload<T>, action: FinishFetchActionType<T, *>
 ): Payload<T> => {
```

Comment:
Is there a reason why you didn't remove the typing for `T` from the `finishFetchReducer`? It don't quite get why `T` actually matters here...

Reply:
It is necessary to indicate that the type of oldPayload is the same as the type of the return value. If we would just use `PayloadDataType`, the value of oldPayload could e.g. be `Array<EventModel>` and the return value `<Array<OfferModel>` which are both of type `PayloadDataType`.